package.json

```diff
@@ -32,13 +32,15 @@
     "babel-core": "^6.26.0",
     "babel-eslint": "^8.2.2",
     "babel-preset-env": "^1.6.1",
+    "babel-preset-flow": "^6.23.0",
     "es6-template-string-loader": "^2.0.0",
```

Comment:
Can we remove this plugin? Its for webpack

Reply:
Done